src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/service/DetEmailService.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.iacasenotificationsapi.domain.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class DetEmailService {
+    private String emailAddress;
+
+    public DetEmailService(@Value("${detUnrepresentedEmailAddress}") String emailAddress) {
+        this.emailAddress = emailAddress;
+    }
+
+    public String getEmail() {
```

Comment:
This email is for ADA cases which are unrepresented. So please change the variable to adaDetEmailAddress. Also method name to getAdaDetEmailAddress. It will help to add some comments on top pf method like (Javadoc) to say - this method returns DET Email Address for ADA Unrepresented cases.

Reply:
det there does not mean detained but Detention Engagement Team. I think this is ok or maybe the full name could be used in stead for clarity.